crates/biome_ungrammar/LICENSE

```diff
@@ -0,0 +1,25 @@
+MIT License
+
+biome_ungrammar is a fork of Ungrammar (https://github.com/rust-analyzer/ungrammar).
+Ungrammar is licensed under the MIT or Apache-2.0 License.
+The copyright holder of Ungrammar is Aleksey Kladov.
```

Comment:
Since Ungrammar doesn't actually have a LICENSE file, this is not _truly_ known, but they are listed as the Author in the Cargo.toml where the license is declared, so I believe it to be as accurate as possible here.